cluster/cluster.go

```diff
@@ -51,7 +51,7 @@ type Cluster interface {
 
 	// Return some info about the cluster, like nb or containers / images
 	// It is pretty open, so the implementation decides what to return.
-	Info() [][2]string
+	Info() ([][]string, int64, int64)
```

Comment:
I agree, and `cluster.CPUs()` it forces us to add 2 new method to the interface.